rave-test/src/main/java/com/uber/rave/DoubleCreator.java

```diff
@@ -26,11 +26,6 @@
  */
 public class DoubleCreator extends ObjectCreator<Double> {
 
-
-    public DoubleCreator(long... validValues) {
```

Comment:
This seems unrelated

Reply:
This wasn't used and stopped working since intDef can no longer take `long` values so I deleted it.

Reply:
How did it stop working if it wasn't used?

Reply:
Compile time breaking change. `AnnotationSpecs.setIntDef` no longer takes long values.

Reply:
So was is `This wasn't used` or `compile time breaking change`? I'm a little confused at this point, especially since you added `LongDef` back now and presumably would use this?

Reply:
> So was is This wasn't used or compile time breaking change?

Both. We had to change AnnotationSpecs.setIntDef to only accept ints which broke this constructor at compile time. Since the constructor was unused but causing compilation to fail I opted just to remove it.